### ISSUE TITLE
p-confirm-dialog: aria-label of Close Button is not set

### ISSUE THREAD

**Issue by StefanTieber** [primefaces/primeng#18164]
### Accessibility Issue with the Close Button

The close button in the ConfirmDialog does not have an aria-label or accessible text by default. While the documentation mentions the closeAriaLabel property, setting it does not apply the aria-label to the close button.

### Example:
`<p-confirm-dialog closeAriaLabel="close" />`

### Expected Behavior:
The aria-label should appear on the close button, ensuring accessibility compliance (e.g., WCAG standards).

### Actual Behavior:
The aria-label is missing from the close button, resulting in accessibility flags during audits.

### Note:
This issue is a duplicate of https://github.com/primefaces/primeng/issues/17404
This is because the linked PR **did not fix** the OPs problem.

### Conjecture:
p-confirm-dialog uses p-dialog internally. However, the closeAriaLabel is never passed to the p-dialog. Maybe there is something I'm not seeing, but I suspect this is the cause. Similarly, the closeButtonProps one can set in ConfirmationService.confirm() don't seem to make it to the close button.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-gbpwkjgt?file=src%2Fapp%2Fapp.component.ts

### Environment

Win 10
Angular dev server
Firefox

But should not matter

### Angular version

19.2.7

### PrimeNG version

v19

### Steps to reproduce the behavior

Set closeAriaLabel on p-confirm-dialog.
Trigger dialog via confirmationService.confirm().

### Expected behavior

aria-label is set on the close button of the dialog
